src/components/http/HttpRequest.brs

```diff
@@ -34,7 +34,7 @@ function HttpRequest(options as Object, httpInterceptors = [] as Object) as Obje
 
   ' @constructor
   ' @param {Object} m - Instance reference
-  _constructor = function (m as Object) as Object
+  _constructor = function(m as Object) as Object
```

Comment:
This is gonna be inconsistent with the standard we chose. Anonymous functions should have space before `(`.  Check out other entities. Why such change?